test/units/galaxy/test_collection.py

```diff
@@ -186,6 +186,25 @@ def manifest(manifest_info):
             yield fake_file, sha256(b_data).hexdigest()
 
 
+class MockHTTPResponses:
+    def __init__(self, responses: list[tuple[bytes, int | None]]):
```

Comment:
incomplete annotation
```suggestion
    def __init__(self, responses: list[tuple[bytes, int | None]]) -> None:
```